contracts/DCAPermissionsManager/DCAPermissionsManager.sol

```diff
@@ -185,6 +220,16 @@ contract DCAPermissionsManager is ERC721, EIP712, Governable, IDCAPermissionMana
     return nftDescriptor.tokenURI(hub, _tokenId);
   }
 
+  function _encode(PositionPermissions[] calldata _permissions) internal pure returns (bytes memory _result) {
+    for (uint256 i; i < _permissions.length; i++) {
+      _result = bytes.concat(_result, keccak256(_encode(_permissions[i])));
+    }
```

Comment:
```suggestion
    for (uint256 i = 0; i < _permissions.length;) {
      _result = bytes.concat(_result, keccak256(_encode(_permissions[i])));
      unchecked {
        i++;
      }
    }
```
Trying to add consistency, specially on the `uint256 = 0` that we are using on L201

Reply:
Already done in #232